### ISSUE TITLE
update chat settings websocket event

### ISSUE THREAD

**Issue by thePlebDev** [thePlebDev/Clicker#1065]
# Proposed change
- I need to be able to subscribe to the websocket endpoint and get the instant updates about changes to the chat settings data

# Why is this important
- This is a needed feature of the modView features

# Additional context
- [HERE](https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types/#channelchat_settingsupdate) is the websocket susbscription type for getting chat settings update

**Comment by thePlebDev** [comment]
- Here is some data that can be parsed for testing:
```
{"metadata":{"message_id":"rZ-HByBPf2Mc7NfYSgR5mlsf0jHsNdZ9VKKb313Lbjw=","message_type":"notification","message_timestamp":"2024-04-14T23:53:59.928981523Z","subscription_type":"channel.chat_settings.update","subscription_version":"1"},"payload":{"subscription":{"id":"46440d9c-e754-47be-840a-943c1250a222","status":"enabled","type":"channel.chat_settings.update","version":"1","condition":{"broadcaster_user_id":"520593641","user_id":"946933663"},"transport":{"method":"websocket","session_id":"AgoQvN_TbZDMRr6yuimMnf6ZHRIGY2VsbC1i"},"created_at":"2024-04-14T23:52:23.445062985Z","cost":0},"event":{"broadcaster_user_id":"520593641","broadcaster_user_login":"theplebdev","broadcaster_user_name":"theplebdev","emote_mode":false,"follower_mode":false,"follower_mode_duration_minutes":null,"slow_mode":false,"slow_mode_wait_time_seconds":null,"subscriber_mode":false,"unique_chat_mode":false}}}

{"metadata":{"message_id":"4ucSfpP_-fYeADPZl9S9p69fEBrbNx4r3pkwFdhfRQY=","message_type":"notification","message_timestamp":"2024-04-14T23:54:03.472018719Z","subscription_type":"channel.chat_settings.update","subscription_version":"1"},"payload":{"subscription":{"id":"46440d9c-e754-47be-840a-943c1250a222","status":"enabled","type":"channel.chat_settings.update","version":"1","condition":{"broadcaster_user_id":"520593641","user_id":"946933663"},"transport":{"method":"websocket","session_id":"AgoQvN_TbZDMRr6yuimMnf6ZHRIGY2VsbC1i"},"created_at":"2024-04-14T23:52:23.445062985Z","cost":0},"event":{"broadcaster_user_id":"520593641","broadcaster_user_login":"theplebdev","broadcaster_user_name":"theplebdev","emote_mode":false,"follower_mode":false,"follower_mode_duration_minutes":null,"slow_mode":false,"slow_mode_wait_time_seconds":null,"subscriber_mode":true,"unique_chat_mode":false}}}

{"metadata":{"message_id":"MuSsMeCv0TpBuNlPZvfIMqVCV_AlhjesIgKNLsohdGc=","message_type":"notification","message_timestamp":"2024-04-14T23:54:09.751358664Z","subscription_type":"channel.chat_settings.update","subscription_version":"1"},"payload":{"subscription":{"id":"46440d9c-e754-47be-840a-943c1250a222","status":"enabled","type":"channel.chat_settings.update","version":"1","condition":{"broadcaster_user_id":"520593641","user_id":"946933663"},"transport":{"method":"websocket","session_id":"AgoQvN_TbZDMRr6yuimMnf6ZHRIGY2VsbC1i"},"created_at":"2024-04-14T23:52:23.445062985Z","cost":0},"event":{"broadcaster_user_id":"520593641","broadcaster_user_login":"theplebdev","broadcaster_user_name":"theplebdev","emote_mode":false,"follower_mode":false,"follower_mode_duration_minutes":null,"slow_mode":false,"slow_mode_wait_time_seconds":null,"subscriber_mode":false,"unique_chat_mode":false}}}

{"metadata":{"message_id":"g2ykpuyN1her3x8waCD3FDpqLwu4ZlLhDyEt0mOSjyQ=","message_type":"notification","message_timestamp":"2024-04-14T23:54:12.859251705Z","subscription_type":"channel.chat_settings.update","subscription_version":"1"},"payload":{"subscription":{"id":"46440d9c-e754-47be-840a-943c1250a222","status":"enabled","type":"channel.chat_settings.update","version":"1","condition":{"broadcaster_user_id":"520593641","user_id":"946933663"},"transport":{"method":"websocket","session_id":"AgoQvN_TbZDMRr6yuimMnf6ZHRIGY2VsbC1i"},"created_at":"2024-04-14T23:52:23.445062985Z","cost":0},"event":{"broadcaster_user_id":"520593641","broadcaster_user_login":"theplebdev","broadcaster_user_name":"theplebdev","emote_mode":false,"follower_mode":false,"follower_mode_duration_minutes":null,"slow_mode":true,"slow_mode_wait_time_seconds":30,"subscriber_mode":false,"unique_chat_mode":false}}}

{"metadata":{"message_id":"6Zx0AR-fAVz53qLmBDqD_B86oG3Vs9Q06GLKx34geBQ=","message_type":"notification","message_timestamp":"2024-04-14T23:54:16.891622111Z","subscription_type":"channel.chat_settings.update","subscription_version":"1"},"payload":{"subscription":{"id":"46440d9c-e754-47be-840a-943c1250a222","status":"enabled","type":"channel.chat_settings.update","version":"1","condition":{"broadcaster_user_id":"520593641","user_id":"946933663"},"transport":{"method":"websocket","session_id":"AgoQvN_TbZDMRr6yuimMnf6ZHRIGY2VsbC1i"},"created_at":"2024-04-14T23:52:23.445062985Z","cost":0},"event":{"broadcaster_user_id":"520593641","broadcaster_user_login":"theplebdev","broadcaster_user_name":"theplebdev","emote_mode":false,"follower_mode":false,"follower_mode_duration_minutes":null,"slow_mode":false,"slow_mode_wait_time_seconds":null,"subscriber_mode":false,"unique_chat_mode":false}}}

{"metadata":{"message_id":"n_jhJ7PnsVy_EdOj6vlgyieQSroBF4YHsV-RsOXUzfk=","message_type":"notification","message_timestamp":"2024-04-14T23:54:25.886551884Z","subscription_type":"channel.chat_settings.update","subscription_version":"1"},"payload":{"subscription":{"id":"46440d9c-e754-47be-840a-943c1250a222","status":"enabled","type":"channel.chat_settings.update","version":"1","condition":{"broadcaster_user_id":"520593641","user_id":"946933663"},"transport":{"method":"websocket","session_id":"AgoQvN_TbZDMRr6yuimMnf6ZHRIGY2VsbC1i"},"created_at":"2024-04-14T23:52:23.445062985Z","cost":0},"event":{"broadcaster_user_id":"520593641","broadcaster_user_login":"theplebdev","broadcaster_user_name":"theplebdev","emote_mode":false,"follower_mode":true,"follower_mode_duration_minutes":33,"slow_mode":false,"slow_mode_wait_time_seconds":null,"subscriber_mode":false,"unique_chat_mode":false}}}

{"metadata":{"message_id":"ZiAfPJJnoWdrlMTHUdsOsQQkUdlNu31rnomyXSNqniA=","message_type":"notification","message_timestamp":"2024-04-14T23:54:34.470892023Z","subscription_type":"channel.chat_settings.update","subscription_version":"1"},"payload":{"subscription":{"id":"46440d9c-e754-47be-840a-943c1250a222","status":"enabled","type":"channel.chat_settings.update","version":"1","condition":{"broadcaster_user_id":"520593641","user_id":"946933663"},"transport":{"method":"websocket","session_id":"AgoQvN_TbZDMRr6yuimMnf6ZHRIGY2VsbC1i"},"created_at":"2024-04-14T23:52:23.445062985Z","cost":0},"event":{"broadcaster_user_id":"520593641","broadcaster_user_login":"theplebdev","broadcaster_user_name":"theplebdev","emote_mode":false,"follower_mode":false,"follower_mode_duration_minutes":null,"slow_mode":false,"slow_mode_wait_time_seconds":null,"subscriber_mode":false,"unique_chat_mode":false}}}

```

**Comment by thePlebDev** [comment]
- closed with #1067
